packages/datadog-instrumentations/src/mquery.js

```diff
@@ -1,15 +1,14 @@
 'use strict'
 
+const dc = require('dc-polyfill')
 const {
   channel,
-  addHook,
-  AsyncResource
+  addHook
 } = require('./helpers/instrument')
 const shimmer = require('../../datadog-shimmer')
 
-const prepareCh = channel('datadog:mquery:filter:prepare')
-const startCh = channel('datadog:mquery:filter:start')
-const finishCh = channel('datadog:mquery:filter:finish')
+const prepareCh = channel('tracing:datadog:mquery:filter:prepare')
```

Comment:
Please don't use the `tracing:` prefix for non-tracing-channel events.

Reply:
@Qard can you take a look at the changes? thkx